internal/extproc/processor.go

```diff
@@ -184,8 +204,13 @@ func (p *Processor) ProcessResponseBody(_ context.Context, body *extprocv3.HttpB
 		return &extprocv3.ProcessingResponse{Response: &extprocv3.ProcessingResponse_ResponseBody{}}, nil
 	}
 
-	headerMutation, bodyMutation, tokenUsage, err := p.translator.ResponseBody(p.responseHeaders, br, body.EndOfStream)
+	headerMutation, bodyMutation, tokenUsage, err := p.translator.ResponseBody(p.responseHeaders, br, body.EndOfStream, p.backendName, p.modelName)
 	if err != nil {
+		if p.backendName != "" && p.modelName != "" {
+			p.metrics.RequestsTotal.WithLabelValues(p.backendName, p.modelName, "error").Inc()
```

Comment:
I think we should track the error count at the higher level here 
https://github.com/envoyproxy/ai-gateway/blob/9d4fc925b81249231f2b0a5ba394ed8e8536e336/internal/extproc/server.go#L185

Reply:
`processMsg` doesn't have model or backend names yet.